IQSS/dataverse-frontend
# 174 - Refine UI styling Files Tab

## What this PR does / why we need it:

This PR addresses the issue of CSS styling inconsistencies in the Files Tab. It ensures that the styles for the Files Tab are consistent with the JSF UI.

## Which issue(s) this PR closes:

- Closes #174

## Special notes for your reviewer:
Some changes where requested in this other PR 
- https://github.com/IQSS/dataverse-frontend/pull/162

## Suggestions on how to test this:

To test the changes introduced by this PR, follow these steps:

1. Install the necessary dependencies using `npm install`.
3. Build the design system with `cd packages/design-system && npm run build`.
4. Navigate back to the root directory with `cd ../../`.
5. Start Storybook with `npm run storybook`.
6. Visit the Dataset Page in Storybook and inspect the Files Tab.

Ensure that the Files Tab elements are displayed consistently and that the styles have been improved according to the changes made in this PR. Compare the display to the previous state

<img width="1163" alt="column_limiter" src="https://github.com/IQSS/dataverse-frontend/assets/17010447/97efe988-ae8d-43ba-b586-4e96729519ba">

- 1) There is a line between columns that looks like introduced by default. This does not exist in JSF and can be confusing for the user.

- 2) The button group of actions looks like two different buttons. The top and bottom lines of the button group should be straight as in the default react bootstrap component (https://react-bootstrap.netlify.app/docs/components/button-group)

- 3) Vertically align in the middle the text and checkbox of the header, the action buttons and the checkbox of the table rows
 ![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/ed6891fb-e504-4049-a169-7e248218f229)
![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/d3f55f9b-a626-4f7c-9caa-b29f2cd15820)

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:
The above images represent the requested changes

## Is there a release notes update needed for this change?:

No

## Additional documentation:
